build failure in cvss crate with v4/v4 features

When building the cvss crate like this it fails:

```
$ cargo build --no-default-features --features v4
   Compiling cvss v2.2.0 (/tmp/r/cvss-2.2.0)
warning: unused import: `Severity`
  --> src/cvss.rs:10:5
   |
10 |     Severity,
   |     ^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` (part of `#[warn(unused)]`) on by default

error[E0433]: failed to resolve: use of unresolved module or unlinked crate `v3`
   --> src/cvss.rs:157:8
    |
157 |     V3(v3::MetricType),
    |        ^^ use of unresolved module or unlinked crate `v3`
    |
help: to make use of source file src/v3.rs, use `mod v3` in this file to declare the module
   --> src/lib.rs:28:1
    |
 28 + mod v3;
    |

For more information about this error, try `rustc --explain E0433`.
warning: `cvss` (lib) generated 1 warning
error: could not compile `cvss` (lib) due to 1 previous error; 1 warning emitted
```

And looking at the code https://github.com/rustsec/rustsec/blob/main/cvss/src/cvss.rs#L157 it seems like `MetricType` lacks a #[cfg(feature = "v4")] or #[cfg(feature = "v3")] annotation that seems to be present everywhere else.

--- a/cvss/src/cvss.rs
+++ b/cvss/src/cvss.rs
@@ -154,7 +154,9 @@ impl fmt::Display for Cvss {
 #[non_exhaustive]
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum MetricType {
+    #[cfg(feature = "v3")]
     V3(v3::MetricType),
+    #[cfg(feature = "v4")]
     V4(v4::MetricType),
 }
 
@@ -162,7 +164,9 @@ impl MetricType {
     /// Get the name of this metric (i.e. acronym)
     pub fn name(self) -> &'static str {
         match self {
+            #[cfg(feature = "v3")]
             Self::V3(m) => m.name(),
+            #[cfg(feature = "v4")]
             Self::V4(m) => m.name(),
         }
     }
@@ -170,7 +174,9 @@ impl MetricType {
     /// Get a description of this metric.
     pub fn description(self) -> &'static str {
         match self {
+            #[cfg(feature = "v3")]
             Self::V3(m) => m.description(),
+            #[cfg(feature = "v4")]
             Self::V4(m) => m.description(),
         }
     }

